fuel-client/src/client/schema/snapshots/fuel_gql_client__client__schema__tx__tests__transparent_transaction_by_id_query_gql_output.snap

```diff
@@ -4,83 +4,10 @@ expression: operation.query
 ---
 query Query($_0: TransactionId!) {
   transaction(id: $_0) {
-    gasLimit
-    gasPrice
-    id
-    inputAssetIds
-    inputContracts {
-      id
-    }
-    inputs {
-      __typename
-      ... on InputCoin {
-        utxoId
-        owner
-        amount
-        assetId
-        txPointer
-        witnessIndex
-        maturity
-        predicate
-        predicateData
-      }
-      ... on InputContract {
-        utxoId
-        balanceRoot
-        stateRoot
-        txPointer
-        contract {
-          id
-        }
-      }
-      ... on InputMessage {
-        messageId
-        sender
-        recipient
-        amount
-        nonce
-        witnessIndex
-        data
-        predicate
-        predicateData
-      }
-    }
-    isScript
-    outputs {
-      __typename
-      ... on CoinOutput {
-        to
-        amount
-        assetId
-      }
-      ... on ContractOutput {
-        inputIndex
-        balanceRoot
-        stateRoot
-      }
-      ... on MessageOutput {
-        recipient
-        amount
-      }
-      ... on ChangeOutput {
-        to
-        amount
-        assetId
-      }
-      ... on VariableOutput {
-        to
-        amount
-        assetId
-      }
-      ... on ContractCreated {
-        contract {
-          id
-        }
-        stateRoot
-      }
+    rawPayload
```

Comment:
Hmm, I'm not sure why do we have changes here=D

Reply:
Freshly generated .snap file based on the new settings. I don't know why he generated it like this, but I could theorize :D

Reply:
The transparent types mean we use the typed fields rather than canonical "rawPayload" bytes. By moving this query to use the "rawPayload", it is no longer transparent and we lose our test coverage of these fields on the graphql api.